src/Service/SF1500/SF1500.php

```diff
@@ -397,88 +388,67 @@ public function getEnhedNavnOgOverordnetOrganisationsId($organisationEnhedsId, $
-     * Returns null if no manager exists.
+     * Returns empty array if no manager exists.
+     * @throws SF1500Exception|SAMLTokenException
      */
-    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId)
+    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId): array
     {
-        $token = $this->fetchSAMLToken();
-
         $organisationFunktionsId = $this->getOrganisationFunktionerFromUserId($userId);
 
-        if (null === $organisationFunktionsId) {
-            throw new SF1500Exception(sprintf('Could not find any organisation funktioner for user %s', $userId));
+        if (empty($organisationFunktionsId)) {
+            throw new SF1500Exception(sprintf('Cannot find any organisation funktioner for user %s', $userId));
         }
 
         $managers = [];
-        if (is_array($organisationFunktionsId)) {
-            foreach ($organisationFunktionsId as $id) {
-                $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($id, $managerFunktionsTypeId, $token);
 
-                if (null !== $managerInfo) {
-                    $managers = array_merge($managers, $managerInfo);
-                }
-            }
-        } else {
-            $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($organisationFunktionsId, $managerFunktionsTypeId, $token);
+        foreach ($organisationFunktionsId as $id) {
+            $managerInfo = $this->getManagerBrugerAndFunktionsIdFromFunktionsId($id, $managerFunktionsTypeId);
 
-            if (null !== $managerInfo) {
-                $managers = $managerInfo;
+            if (!empty($managerInfo)) {
+                $managers = array_merge($managers, $managerInfo);
             }
         }
 
-        if (empty($managers)) {
-            return null;
-        }
-
         return $managers;
     }
 
     /**
      * Fetches bruger and organisation funktions id for managers from funktions id.
-     * Returns null if no manager exists
+     * Returns empty array if no manager exists
      * @throws SF1500Exception
      */
-    private function getManagerBrugerAndFunktionsIdFromFunktionsId($funktionsId, $managerFunktionsTypeId, $token)
+    private function getManagerBrugerAndFunktionsIdFromFunktionsId($funktionsId, $managerFunktionsTypeId): array
     {
         $orgId = $this->getOrganisationEnhed($funktionsId, true);
 
         // If current user is manager, start searching one level up in organisation tree.
         if ($managerFunktionsTypeId === $this->getOrganisationFunktionsTypeFromOrganisationFunktion($funktionsId)) {
-            $orgId = $this->getOverordnetOrganisatonEnhedId($orgId, $token);
+            $orgId = $this->getOverordnetOrganisatonEnhedId($orgId);
 
             if (null === $orgId) {
                 // We have reached the top
-                return null;
+                return [];
             }
         }
```

Comment:
What happens if the first `$orgId` (in line 422) is `null`? Consider rewriting to

```php
if (null !== $orgId && $managerFunktionsTypeId === …) { … }

if (null === $orgId) {
  // We have reached the top
  return [];
}
```

i.e. move the early return outside the `if` block.

Reply:
Fixed.